Modules/ThirdParty/TIFF/CMakeLists.txt

```diff
@@ -8,29 +8,20 @@ if(ITK_USE_SYSTEM_TIFF)
   find_package(TIFF REQUIRED)
   set(ITKTIFF_INCLUDE_DIRS
     ${ITKTIFF_BINARY_DIR}/src
+    ${TIFF_INCLUDE_DIRS}
     )
   set(ITKTIFF_SYSTEM_INCLUDE_DIRS
-    ${TIFF_INCLUDE_DIR}
     )
   set(ITKTIFF_LIBRARIES "${TIFF_LIBRARIES}")
   set(ITKTIFF_NO_SRC 1)
 
-  # Check availability of different methods to access TIFF's field
-  # data structure.
-  include(CheckTypeSize)
-  set(CMAKE_REQUIRED_INCLUDES ${ITKTIFF_SYSTEM_INCLUDE_DIRS})
-  set(CMAKE_EXTRA_INCLUDE_FILES "tiffio.h")
-  check_type_size(TIFFFieldReadCount ITK_TIFF_HAS_TIFFFieldReadCount)
```

Comment:
These are in a ".h.in" file with a `#cmakedefine`. That file appears to be missing from this PR.